x/docs/md/cli/mac.md

```diff
@@ -1,3 +1,32 @@
+### Methods of Installing Exercism CLI
+
+  - [With Homebrew](#installing-with-homebrew)
+  - [Without Homebrew](#installing-without-homebrew)
+
+Homebrew is a package manager for OS X which installs the stuff that apple
+didn't.
+
+Find out if you have homebrew installed via the terminal.
+
+You can open a terminal using Spotlight by pressing <command><spacebar> and
+then type 'terminal' in the space provided.
+
+Type in the command:
```

Comment:
Just to make sure this is not confused with the `Spotlight` thing above, replace this line with `On the command prompt, type in the command:`.